dbt/adapters/impala/relation.py

```diff
@@ -37,10 +38,27 @@ class ImpalaRelation(BaseRelation):
     include_policy: ImpalaIncludePolicy = ImpalaIncludePolicy()
     quote_character: str = None
     information: str = None
+
+    def __post_init__(self):
+        if (self.type):
+            tracker.track_usage({
+                "event_type": "dbt_impala_model_access",
+                "model_name": self.render(),
+                "model_type": self.type
```

Comment:
Add a key value pair for default strategy incremental_strategy to be consistent with L53-61

Reply:
added as empty string